heartpy/heartpy.py

```diff
@@ -294,6 +295,23 @@ def process(hrdata, sample_rate, windowsize=0.75, report_time=False,
                                          desired_sample_rate=high_precision_fs, working_data=working_data)
 
     working_data = calc_rr(working_data['peaklist'], sample_rate, working_data=working_data)
+    
+    # Obtain Mode
+    mode_v = mode(working_data['RR_list'])
```

Comment:
Stupid question, I dont know this library very well yet. but what is the unit of the RR-intervals in  `working_data['RR_list']`. This has to be in seconds for the equation to work properly.

Reply:
I'll need to take a look as it was quite a while from when I issued this request. I forgot if the user can control the unit or if it is hardcoded. Thanks!